URBNSwiftCarousel/Classes/URBNCarouselZoomableCellCollectionViewCell.swift

```diff
@@ -6,3 +6,52 @@
 //
 //
 
+import UIKit
+
+public class URBNCarouselZoomableCell: UICollectionViewCell, UIScrollViewDelegate {
+    public var scrollView: UIScrollView?
```

Comment:
I don't think this needs to be optional. it also looks like it can be a constant